src/config/settings.ts

```diff
@@ -1,9 +1,24 @@
 import type { Settings } from "../types"
+import { BlockchainEnum } from "../types/deposit"
 
 export let settings: Settings = {
   providerIds: [],
   defuseContractId: "defuse-alpha.near",
   swapExpirySec: 600, // 10 minutes
+  blockchains: {
+    near: {
+      name: BlockchainEnum.NEAR,
+      icon: "/static/icons/network/near.svg",
+    },
+    ethereum: {
+      name: BlockchainEnum.ETHEREUM,
+      icon: "/static/icons/network/ethereum.svg",
+    },
+    base: {
+      name: BlockchainEnum.BASE,
+      icon: "/static/icons/network/base.svg",
+    },
+  },
```

Comment:
I don't it is "settings", it is just a list of blockchains.

Reply:
Well, we probably want to set up a deposit network at the SDK level, and maybe not just at the deposit level.

Reply:
Then it should be a list of strings then.